shared-libs/cht-datasource/src/person.ts

```diff
@@ -44,6 +61,32 @@ export namespace v1 {
       };
     };
 
+  const getPeople = <T>(
```

Comment:
Just a random final thought here.  I cannot come up with a scenario in the near future where we will be re-using this function.  It might make sense to just combine this back in with `getPage`.  Will definitely simplify things a bit.

(Also, with how we had decided to go with this documentation, it really is going to be single-purpose...)